torch/csrc/cuda/Module.cpp

```diff
@@ -892,6 +892,11 @@ static void registerCudaDeviceProperties(PyObject* module) {
       .def_readonly(
           "max_threads_per_multi_processor",
           &cudaDeviceProp::maxThreadsPerMultiProcessor)
+#if !USE_ROCM
+      // NVIDA only property
+      .def_readonly(
+          "regsPerMultiprocessor", &cudaDeviceProp::regsPerMultiprocessor)
```

Comment:
```suggestion
          "regs_per_multiprocessor", &cudaDeviceProp::regsPerMultiprocessor)
```
to match the style above